Sources/VLCAlertControllerExtension.swift

```diff
@@ -0,0 +1,109 @@
+/*****************************************************************************
+ * VLCAlertControllerExtension.swift
+ * VLC for iOS
+ *****************************************************************************
+ * Copyright (c) 2018 VideoLAN. All rights reserved.
+ * $Id$
+ *
+ * Authors: Pierre Sagaspe <pierre.sagaspe # me.com>
+ *
+ * Refer to the COPYING file of the official project for license.
+ *****************************************************************************/
+
+typealias UIAlertCompletionBlock = (UIAlertController, Int) -> Void
+typealias UIAlertTextFieldCompletionBlock = (UIAlertController, Int, String) -> Void
+
+struct buttonKeys {
+    static var blocksCancelButtonIndex: Int = 0
+    static var blocksOtherButtonIndex: Int = 1
+    static var blocksDestructiveButtonIndex: Int = 2
+}
+
+@objc extension UIAlertController {
+
+    @objc static func showAlertInViewController( _ viewController: UIViewController,
+                                                            title: String,
+                                                          message: String? = nil,
+                                                cancelButtonTitle: String,
+                                                otherButtonTitles: String? = nil,
```

Comment:
If I it understand correctly, so far there support for an individual other button, should this parameter name be renamed to singular?